### DISCUSSION TITLE
[IntelliSense] Detecting duplicates doesn't work with cx

### DISCUSSION THREAD

**Discussion by lucasrabiec** [tailwindlabs/tailwindcss#9370] [Help]
I am using classix (cx) with tailwind. I also have installed Tailwind CSS Intellisense. I orgnized my tailwind classes with cx and all futures of tailwind intellisense works but detecting duplicates not at all. I added this to config:
```ts
"tailwindCSS.experimental.classRegex": [
  ["cx\\(([^)]*)\\)", "'([^']*)'"]
],
```
but it doesn't help, it detects duplicates in same line but not between all of them.
Any idea?

![image](https://user-images.githubusercontent.com/9741188/191111217-eeec3160-a767-4c5c-81ad-4cf03a42cff0.png)

**Comment by nonameolsson** [comment]
I encountered this as well. Would love to find a way to solve this.
